src/lesson5/javaBeginnerChapter2/Task21.java

```diff
@@ -0,0 +1,59 @@
+package lesson5.javaBeginnerChapter2;
+
+import java.util.Scanner;
+
+//Напишите три цикла выполняющих многократное сложение строк, один с
+//помощью оператора сложения и String, другой с помощью StringBuilder и метода
+//append, а также аналогино для StringBuffer. Сравните скорость их выполнения.
+public class Task21 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
```

Comment:
Of course I expect that should enter text only once and get result from all cycles.